tests/unit/AgentBuilderTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+use AG\ElasticApmLaravel\Agent;
+use AG\ElasticApmLaravel\AgentBuilder;
+use AG\ElasticApmLaravel\Collectors\SpanCollector;
+use AG\ElasticApmLaravel\Exception\MissingAppConfigurationException;
+use Codeception\Test\Unit;
+use Illuminate\Config\Repository as Config;
+
+class AgentBuilderTest extends Unit
+{
+    use Codeception\AssertThrows;
+
+    /** @var AgentBuilder */
+    private $builder;
+
+    /** @var SpanCollector */
+    private $collector;
+
+    protected function _before(): void
+    {
+        $this->builder = new AgentBuilder();
+        $this->collector = Mockery::mock(SpanCollector::class);
+    }
+
+    public function testMissingConfigurationException(): void
+    {
+        self::assertThrows(MissingAppConfigurationException::class, function () {
+            $this->builder->build();
+        });
```

Comment:
This can be done like this, without a need for an extra package:

```suggestion
        $this->expectException(MissingAppConfigurationException::class);
        $this->builder->build();
```

Reply:
Oh nice, I will remove the extra package then.